_docs/guides/best-practices.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Best Practices 
```

Comment:
I think it would be helpful to provide some scenarios that we've seen with deployments. For example 'inventory with IAM Explain' - what is the actual recommendation of such a deployment. How about 'inventory + scanner' etc. The current way the document is setup there is still quite some transfer thinking required.